submissions/LofeWilter/HTML-CSS-Popup/index.html

```diff
@@ -0,0 +1,160 @@
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/mail.png" alt="Mail">
+                        <figcaption>Mail</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/news.png" alt="News">
+                        <figcaption>News</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/maps.png" alt="Map">
+                        <figcaption>Map</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/files.png" alt="Files">
+                        <figcaption>Files</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/contacts.png" alt="Contacts">
+                        <figcaption>Contacts</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/tanslation.png" alt="Translate">
+                        <figcaption>Translate</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/videos.png" alt="Videos">
+                        <figcaption>Videos</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/search.png" alt="Search">
+                        <figcaption>Search</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/photos.png" alt="Photo">
+                        <figcaption>Photo</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/apps.png" alt="Apps">
+                        <figcaption>Apps</figcaption>
+                    </figure>
+                </a>
+                <a href="">
+                    <figure class="img">
+                        <img src="assets/documents.png" alt="Document">
+                        <figcaption>Document</figcaption>
+                    </figure>
+                </a>
+            </div>
+            <input type="checkbox" id="evenmore" class="secondCheckbox">
+            <label for="evenmore" class="hide">
+                <div class="more">
```

Comment:
Element `div` not allowed as child of element `label` in this context. (Suppressing further errors from this subtree.)